pkg/tenancy/tenancy.go

```diff
@@ -94,3 +102,11 @@ func getTenantFromCertificate(r *http.Request, certTenantField string) (string,
 
 	return tenant, nil
 }
+
+func GetTenantFromGRPCMetadata(ctx context.Context) (string, error) {
+	md, ok := metadata.FromIncomingContext(ctx)
+	if !ok || len(md.Get(DefaultTenantHeader)) == 0 {
+		return DefaultTenant, fmt.Errorf("could not get tenant from grpc metadata, using default: %s", DefaultTenantHeader)
```

Comment:
We usually don't use `fmt.Errorf` in Thanos (there's a pending lint rule in Makefile). 
```suggestion
		return DefaultTenant, errors.Errorf("could not get tenant from grpc metadata, using default: %s", DefaultTenantHeader)
```

Reply:
Fixed :+1: